ci/pipeline

```diff
@@ -38,11 +39,12 @@ def compileAndTest(logFileName: String): Unit = utils.stage("Compile and Test")
   run("sbt", "plugin-interface/compile")
 
   // Check system integraion tests.
-  checkSystemIntegrationTests(logFileName)
+  // checkSystemIntegrationTests(logFileName)
 }
 
 @main
 def checkSystemIntegrationTests(logFileName: String): Unit = {
+  // TODO: (kgs)  There is NO need to kill processes for this!
```

Comment:
I don't understand the TODO. Is the todo to remove the timeout?

So, I've left it because the `runWithTimeout` redirect stdout into a file. The name is probably not accurate anymore. We could introduce a infinite timeout. What do you think?